homework/index.js

```diff
@@ -21,27 +21,117 @@
     parent.appendChild(elem);
     Object.keys(options).forEach(key => {
       const value = options[key];
-      if (key === 'text') {
-        elem.innerText = value;
+      if (key === 'html') {
+        elem.innerHTML = value;
       } else {
         elem.setAttribute(key, value);
       }
     });
     return elem;
   }
 
+  function showRepos(leftDiv, repository) {
+    leftDiv.innerHTML = '';
+    const table = createAndAppend('table', leftDiv);
+    const tableBody = createAndAppend('tbody', table);
+    const repositoryTitle = createAndAppend('tr', tableBody);
+    createAndAppend('td', repositoryTitle, { html: 'Repository:', class: 'label' });
+    createAndAppend('a', repositoryTitle, {
+      target: '_blank',
+      href: repository.html_url,
+      html: repository.name,
+    });
+    if (repository.description !== null) {
+      const description = createAndAppend('tr', tableBody);
+      createAndAppend('td', description, {
+        html: 'Description:',
+        class: 'label',
+      });
+      createAndAppend('td', description, { html: repository.description });
+    }
+    const forks = createAndAppend('tr', tableBody);
+    createAndAppend('td', forks, { html: 'Forks:', class: 'label' });
+    createAndAppend('td', forks, { html: repository.forks });
+    const update = createAndAppend('tr', tableBody);
+    createAndAppend('td', update, { html: 'Updated:', class: 'label' });
+    createAndAppend('td', update, { html: repository.updated_at });
+  }
+
+  function showContributors(rightDiv, repository) {
+    rightDiv.innerHTML = '';
+    createAndAppend('p', rightDiv, {
+      html: 'Contributions',
+      class: 'contributor-head',
+    });
+    fetchJSON(repository.contributors_url, (err, contributors) => {
+      if (err) {
+        createAndAppend('div', rightDiv, {
+          html: err.message,
+          class: 'alert-error',
+        });
+      }
```

Comment:
When there is an error you should not continue with the rest of the code because there is no data to work on. Instead, you should add a `return` statement to and the end of this `if` block to return early from this callback function.